packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -258,4 +318,339 @@ describe('Select', () => {
             expect(getSelectedValueTitle()).toBe('');
         });
     });
+
+    describe('opening and closing dropdown', () => {
+        let element: Select;
+        let connect: () => Promise<void>;
+        let disconnect: () => Promise<void>;
+        let pageObject: SelectPageObject;
+
+        beforeEach(async () => {
+            ({ element, connect, disconnect } = await setup());
+            element.style.width = '200px';
+            element.filterMode = FilterMode.standard;
+            await connect();
+            pageObject = new SelectPageObject(element);
+        });
+
+        afterEach(async () => {
+            await disconnect();
+        });
+
+        const filterModeTestData = [
+            {
+                filter: FilterMode.none,
+                name: 'none'
+            },
+            {
+                filter: FilterMode.standard,
+                name: 'standard'
+            }
+        ];
+        filterModeTestData.forEach(testData => {
```

Comment:
And can you organize the test as parameterized suites?

```ts
filterModeTestData.forEach(testData => {
    describe(`with filterMode = ${testData.name}`, () => {
        it(... /* now each test does not need to say it's filter mode since it is part of the describe block */)
        it(...)
    })

})
```

Reply:
Created #1827.